keyboards/1upkeyboards/1upslider8/config.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2023 Ziptyze
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define I2C1_SCL_PIN        GP11
+#define I2C1_SDA_PIN        GP10
+#define OLED_BRIGHTNESS 128
+#define OLED_FONT_H "keyboards/1upkeyboards/1upslider8/lib/glcdfont.c"
```

Comment:
This should work fine: 
```suggestion
#define OLED_FONT_H "lib/glcdfont.c"
```